scss/bootstrap/components/_checkbox-switch.scss

```diff
@@ -0,0 +1,104 @@
+.checkbox-switch {
+  padding-top: 0 !important;
+  position: relative;
+
+  input {
```

Comment:
i'd suggest to give this element a classname, so you can avoid nesting and flatten the selectors. Something like `.checkbox-switch-input`

Reply:
We decided not to add class to the input.